testing/infra/k8s/base/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+resources:
+  - https://download.elastic.co/downloads/eck/2.4.0/crds.yaml
+  - https://download.elastic.co/downloads/eck/2.4.0/operator.yaml
```

Comment:
I wonder how we'll keep these up to date.

Reply:
Like end users, I don't think it's too much of a problem if we're not always on the latest version. Only if there's some security issue or critical bug, or a new feature that we want to take advantage of.